tests/rbd/krbd_io_handler.py

```diff
@@ -143,7 +150,7 @@ def krbd_io_handler(**kw):
                         )
                         return_flag = 1
                     if operations.get("device_map"):
-                        if rbd.device_unmap(
+                        if rbd.device_map(
```

Comment:
but still command here is `unmap`?

Reply:
yes, you can see i am passing parameter  "unmap" to input of method device_map. 
Based on parameter passed "map" or "unmap" it will do the required action.